mobile/app/(app)/(drawer)/(tabs)/more.tsx

```diff
@@ -1,10 +1,105 @@
-import { View, Text } from "react-native";
+import React, { useState } from "react";
+import { View, XStack } from "tamagui";
+import Card from "../../../../components/Card";
+import { Screen } from "../../../../components/Screen";
+import { Typography } from "../../../../components/Typography";
+import { Icon } from "../../../../components/Icon";
+import { useTranslation } from "react-i18next";
+import * as Linking from "expo-linking";
+import { tokens } from "../../../../theme/tokens";
 
 const More = () => {
+  const { t } = useTranslation("more");
+
+  // TODO: Change these consts
+  const appVersion = "2.0.4";
+  const appLanguage = "English (United States)";
+  const URL = "https://www.google.com/";
+
+  return (
+    <Screen
+      preset="auto"
+      backgroundColor="white"
+      contentContainerStyle={{
+        gap: tokens.space.md.val,
+        paddingHorizontal: tokens.space.md.val,
+        paddingVertical: tokens.space.xl.val,
+      }}
+    >
+      <Section
+        headerText={t("change-language")}
+        subHeaderText={appLanguage}
+        iconName="language"
+        chevronRight={true}
+      ></Section>
+      <Section
+        headerText={t("change-password")}
+        iconName="changePassword"
+        chevronRight={true}
+      ></Section>
+      <Section
+        headerText={t("terms")}
+        iconName="termsConds"
+        chevronRight={true}
+        action={() => {
+          Linking.openURL(URL);
+        }}
+      ></Section>
+      <Section
+        headerText={t("privacy_policy")}
+        iconName="privacyPolicy"
+        chevronRight={true}
+        action={() => {
+          Linking.openURL(URL);
+        }}
+      ></Section>
+      <Section
+        headerText={t("about")}
+        subHeaderText={t("app_version", { value: appVersion })}
+        iconName="aboutVM"
+        chevronRight={true}
+      ></Section>
+      <Section headerText={t("support")} iconName="contactNGO"></Section>
+      <Section headerText={t("feedback")} iconName="feedback"></Section>
+      <Section headerText={t("logout")} iconName="logoutNoBackground"></Section>
+    </Screen>
+  );
+};
+
+interface SectionProps {
+  headerText: string;
+  subHeaderText?: string;
+  iconName: string;
+  chevronRight?: boolean;
+  action?: () => void;
+}
+
+const Section = (props: SectionProps) => {
```

Comment:
I would like this to be called MenuItem, update the props interface also

Reply:
done